odoo/addons/base/models/ir_rule.py

```diff
@@ -143,6 +184,26 @@ def write(self, vals):
         self.clear_caches()
         return res
 
+    def _make_access_error(self, operation, records):
+        _logger.info('Access Denied by record rules for operation: %s on record ids: %r, uid: %s, model: %s', operation, records.ids, self._uid, records._name)
```

Comment:
The log message used to limit the number of IDs being output to 6, to avoid writing a useless list with millions of IDs, wasting both space and time. I imagine that if you have more than a few IDs, the actual list is not terribly relevant.

Reply:
Good point. A possible alternative to just `[:6]` would be [textwrap.shorten](https://docs.python.org/3/library/textwrap.html#textwrap.shorten) with the relevant width (maybe 25 or so), to make it clearer that part of the list was elided.